js/__internal/grids/grid_core/keyboard_navigation/module.ts

```diff
@@ -1931,35 +2337,40 @@ const keyboardNavigationMembers: Partial<import('./module_types').KeyboardNaviga
       $cell = $targetElement.children().not(`.${COMMAND_EXPAND_CLASS}`).first();
     }
     return $cell;
-  },
+  }
+
   _getRowsViewElement() {
     const rowsView = this.getView('rowsView');
     return rowsView && rowsView.element();
-  },
+  }
+
   isKeyboardEnabled() {
     return this.option('keyboardNavigation.enabled');
-  },
+  }
+
   _processCanceledEditCellPosition(rowIndex, columnIndex) {
     if (this._canceledCellPosition) {
-      const isCanceled = this._canceledCellPosition.rowIndex === rowIndex && this._canceledCellPosition.columnIndex === columnIndex;
+      const isCanceled = this._canceledCellPosition.rowIndex === rowIndex
+        && this._canceledCellPosition.columnIndex === columnIndex;
       this._canceledCellPosition = null;
       return isCanceled;
     }
 
     return undefined;
-  },
+  }
+
   updateFocusedRowIndex() {
     const dataController = this._dataController;
     const visibleRowIndex = this.getVisibleRowIndex();
     const visibleItems = dataController.items();
     const lastVisibleIndex = visibleItems.length ? visibleItems.length - 1 : -1;
     const rowIndexOffset = dataController.getRowIndexOffset();
 
-    lastVisibleIndex >= 0 && visibleRowIndex > lastVisibleIndex && this.setFocusedRowIndex(lastVisibleIndex + rowIndexOffset);
-  },
-};
-
-const KeyboardNavigationController: any = core.ViewController.inherit(keyboardNavigationMembers);
+    lastVisibleIndex >= 0
+      && visibleRowIndex > lastVisibleIndex
+      && this.setFocusedRowIndex(lastVisibleIndex + rowIndexOffset);
```

Comment:
Let's change it to ```if``` condition:

```suggestion
    if (lastVisibleIndex >= 0 && visibleRowIndex > lastVisibleIndex) {
      this.setFocusedRowIndex(lastVisibleIndex + rowIndexOffset);
    }
```